backend/clubs/serializers.py

```diff
@@ -96,6 +97,24 @@ def save(self):
         return super().save()
 
 
+class ApplicationCycleSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ApplicationCycle
+        fields = ["id", "name", "start_date", "end_date"]
```

Comment:
Do we need id?

Reply:
Tested it without and it seemed to prevent editing / deletion of fields after page refresh, I assume the numerical id is used somewhere in those operations, but could be wrong.